lib/signature.coffee

```diff
@@ -93,10 +95,13 @@ module.exports = class Signature
 				if parameter.isOptional() and _.isEmpty(value)
 					return result
 
-				result[parameter.getValue()] = value
+				result[parameterValue] = value
 				return result
 
 			if not parameter.isWord() and word?
-				result[parameter.getValue()] = _.parseInt(word) or word
+				if /^\d+$/.test(word)
```

Comment:
It might be worth allowing spaces as well?  Also is there any support for comma-separated numbers or floats (I don't know the context whether either makes sense, but just to think about)